lstm/bmi_lstm.py

```diff
@@ -143,7 +145,7 @@ def __init__(self, cfg: dict[str, typing.Any], output_scaling_factor_cms: float)
         # load training feature scales
         scaler_file = cfg["run_dir"] / "train_data/train_data_scaler.yml"
         with scaler_file.open("r") as fp:
-            train_data_scaler = yaml.safe_load(fp)
+            train_data_scaler = yaml.load(fp, Loader=CLoader)
```

Comment:
I wasn't overly concerned with the performance penalty of using the non-`C`Loader here but I don't think it hurts. If `ngen` creates many instances of this module I could see that adding up. I don't think its worth moving away from the [`SafeLoader`](https://pyyaml.org/wiki/PyYAMLDocumentation) though. Lets use the guarded import suggestion above to try and import the `CSafeLoader` and use that and fallback to `SafeLoader` if `libyaml` is not found.

```suggestion
            train_data_scaler = yaml.load(fp, Loader=SafeLoader)
```